compiler/fir/analysis-tests/testData/resolve/diagnostics/conflictingOverloads.kt

```diff
@@ -59,7 +59,7 @@ class L {
     fun B.foo() {}
 }
 
-fun mest()
+fun mest() {}
```

Comment:
Same here. Not sure what the intention was, though. As per the test name, `conflictingOverloads`, the only conflicting declaration is the class `mest` below.